bootstrap/config.yaml

```diff
@@ -27,15 +32,15 @@ class_data:
   "description": "class_title"
   "allOf": [{"$ref": "https://ns.adobe.com/xdm/data/time-series"}]
 
-mixin_data:
+input_mixin_data:
   "type": "object"
   "title": "mixin_title"
   "description": "mixin_title"
   "meta:intendedToExtend": [{"class_id"}]
   "definitions": {
     "retail": {
       "properties": {
-        "tenant_id": {
+        "_acpmlexploratoryexternal": {
```

Comment:
Is this correct?  Thought things were becoming abstract.

Reply:
Yes, reverted back